frontend/js/app/nginx/proxy/form.ejs

```diff
@@ -105,6 +114,15 @@
                                 </select>
                             </div>
                         </div>
+                        <div class="col-sm-12 col-md-12">
+                            <div class="form-group">
+                                <label class="form-label"><%- i18n('all-hosts', 'ssl-key-type') %></label>
+                                <select name="ssl_key_type" class="form-control custom-select">
```

Comment:
Since this setting only applies to Certificates, it should be a new column on the Certificates table. And then there should be some method of changing it's type from that list. There's currently no "edit certificate" dialog so that's going to have to be a thing.

You've got to remember, most users won't know or care about this new certificate SSL type so we need to give advanced users the option without hindering or overwhelming simpler users.